docs/_posts/2017-02-11-warts.md.dev

```diff
@@ -9,6 +9,14 @@ order: 1
 Here is a list of built-in warts under the
 `org.wartremover.warts` package.
 
+### ArrayEquals
+
+Unlike other collections `==` on arrays and iterators checks reference equality:
```

Comment:
I think you should say "Unlike other *mutable* collection" and update the example to say `mutable.Seq(1) == mutable.Seq(1) // true`

But any sort of built in equality on mutable collections seems problematic.  Should this wart be extended to cover all mutable collections?  (Because: I would argue the only safe sort of built in equality on a mutable thing is reference equality, but of course that sort of equality is "surprising" for mutable collections.)